source/options-storage.ts

```diff
@@ -23,6 +23,7 @@ const migrations = [
 	featureWasRenamed('separate-draft-pr-button', 'one-click-pr-or-gist'), // Merged in May
 	featureWasRenamed('prevent-pr-commit-link-loss', 'prevent-link-loss'), // Merged in May
 	featureWasRenamed('remove-projects-tab', 'remove-unused-repo-tabs'), // Merged in July
+	featureWasRenamed('remove-unused-repo-tabs', 'clean-repo-tabs'), // Merged in July
```

Comment:
We should also probably add a migration from `more-dropdown` to `clean-repo-tabs` because people have been disabling the feature to restore security and insights.